docs/source/markdown/podman-pull.1.md

```diff
@@ -79,12 +79,15 @@ This is a Docker specific option to disable image verification to a Docker
 registry and is not supported by Podman.  This flag is a NOOP and provided
 solely for scripting compatibility.
 
+**--override-arch**=*ARCH*
+Override the machine's default architecture of the image to be pulled. For example, `arm`.
+
 **--override-os**=*OS*
 Use OS instead of the running OS for choosing images
 
-**--override-arch**=*ARCH*
+**--override-variant**=*VARIANT*
 
-Override the machine's default architecture of the image to be pulled. For example, `arm`.
+_VARIANT_ Use _VARIANT_ instead of the running architecture variant for choosing images.  Some images are based off of different variants of the arm architectures, arm/v5 versus arm/v7.
```

Comment:
```suggestion
Use _VARIANT_ instead of the default architecture variant of the container image.  Some images can use multiple variants of the arm architectures, such as arm/v5 and arm/v7.
```